x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/datafeed/DatafeedJob.java

```diff
@@ -318,7 +321,7 @@ private void run(long start, long end, FlushJobAction.Request flushRequest) thro
             try {
                 extractedData = dataExtractor.next();
             } catch (Exception e) {
-                LOGGER.debug("[" + jobId + "] error while extracting data", e);
+                LOGGER.error(new ParameterizedMessage("[{}] error while extracting data", jobId), e);
```

Comment:
now that you made it an error: should it be audited as well?

Reply:
`ProblemTracker` makes sure this is audited